app/services/cache_service.rb

```diff
@@ -61,7 +58,7 @@ def process_translations(listing)
 
     translation_service.cache_listing_translations(
       listing['Id'],
-      strings_to_translate.keys,
+      strings_to_translate.keys.map { |key| "#{key}__c" },
```

Comment:
I don't actually think we need to do this - you can just leave the key as is, then when you're calling it in `getTranslatedString` it can be:

getTranslatedString(
  listing,
  Reserved_community_type_Description,
)